sematic/calculator.py

```diff
@@ -241,13 +246,24 @@ def _wrapper(func_):
                 ).format(_repr_str_iterable(missing_annotations))
             )
 
+        if inline and base_image_tag is not None:
```

Comment:
We could get into situations in which the Resolver image does not have the necessary contents to resolve the inlined future.
Are you counting on users to see this warning and rectify their code / config after the execution fails?
How about ignoring the inline flag and schedule the run externally, and warn that this is happening instead?

Reply:
This is a debatable behavior. In either option, we force something that users did not ask for (force to use the resolver image, or force an external job).
We are not documenting the multiple base image feature anyway, so I don't expect this to be a problem.